### DISCUSSION TITLE
通过安装包安装HertzBeat的文档中使用的JDK版本要求与安装后不匹配

### DISCUSSION THREAD

**Discussion by BKing2020** [apache/hertzbeat#439] [General]
地址：https://hertzbeat.com/docs/start/package-deploy
位置：快速入门-安装包方式部署
问题：
<img width="788" alt="image" src="https://user-images.githubusercontent.com/28869121/201570932-de87617d-1b72-4d34-a3b9-c420b22bfcc3.png">

**Comment by tomsun28** [comment]
has fixed, thanks!
